src/ClaimService/appsettings.json

```diff
@@ -0,0 +1,57 @@
+{
+  "Serilog": {
+    "Using": [],
+    "LevelSwitches": { "$controlSwitch": "Information" },
+    "MinimumLevel": { "ControlledBy": "$controlSwitch" },
+    "Enrich": [
+      "FromLogContext",
+      "WithMachineName",
+      "WithProcessId",
+      "WithThreadId"
+    ],
+    "WriteTo": [
+      {
+        "Name": "Console"
+      },
+      {
+        "Name": "Seq",
+        "Args": {
+          "serverUrl": "http://localhost:9870",
+          "apiKey": "VGVVhxn9hpcroyV5Anrc"
+        }
+      }
+    ]
+  },
+  "ServiceInfo": {
+    "Id": "7E0E84E5-A139-4211-B293-404637F01F48",
+    "Name": "ClaimService"
+  },
+  "AllowedHosts": "*",
+  "RabbitMQ": {
+    //UserService
+    "CheckUserIsAdminEndpoint": "UserService_GetUserAdminStatus",
+    "GetUsersDataEndpoint": "UserService_GetUsersData",
+    "CheckUsersExistenceEndpoint": "UserService_CheckUsersExistence",
```

Comment:
Will you need? 
Same question for DepartmentService and EmailService

Reply:
I need it for some logic, and if some endpoints will be useless for me, I will delete it